scripts/amex_browseros_bridge.py

```
@@ -0,0 +1,543 @@
+            init_timeout=ACTION_TIMEOUT_SECONDS,
+        )
+
+    async def __aenter__(self) -> "FastMCPBrowserOSSession":
+        await asyncio.to_thread(ensure_browseros_mcp, self.endpoint)
+        await self.client.__aenter__()
+        return self
+
+    async def __aexit__(self, *args: object) -> None:
+        await self.client.__aexit__(*args)
+
+    async def call(self, tool_name: str, arguments: dict[str, Any]) -> Any:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `call`

(ANN401)